### PR TITLE
feature - improved login ui

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Login/ForgotPassWordViewController.swift [hunk 1]
```diff
@@ -17,6 +17,8 @@ class ForgotPassWordViewController: ViewController {
 
         view.backgroundColor = UIColor.black
 
+        resetEmailTextField.attributedPlaceholder =
+            NSAttributedString(string: "Email", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/ForgotPassWordViewController.swift]: Define string and color

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 2]
```diff
@@ -16,6 +16,12 @@ class LoginViewController: ViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        userNameTextField.attributedPlaceholder =
+            NSAttributedString(string: "Username", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: Define

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 3]
```diff
@@ -16,6 +16,12 @@ class LoginViewController: ViewController {
...
+
+        passwordTextField.attributedPlaceholder =
+            NSAttributedString(string: "Password", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: Define

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 4]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        emailSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Email", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: .

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 5]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
...
+
+        passwordSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Password", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: .

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 6]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
...
+
+        nameSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Name", attributes: [NSForegroundColorAttributeName: UIColor.RGB(192, 192, 192)])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: .

File: MyApp/View/Controllers/Login/ForgotPassWordViewController.swift [hunk 7]
```diff
@@ -17,6 +17,8 @@ class ForgotPassWordViewController: ViewController {
 
         view.backgroundColor = UIColor.black
 
+        resetEmailTextField.attributedPlaceholder =
+            NSAttributedString(string: "Email", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/ForgotPassWordViewController.swift]: Define string "Email"

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 8]
```diff
@@ -16,6 +16,12 @@ class LoginViewController: ViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        userNameTextField.attributedPlaceholder =
+            NSAttributedString(string: "Username", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: Define "Username"

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 9]
```diff
@@ -16,6 +16,12 @@ class LoginViewController: ViewController {
...
+
+        passwordTextField.attributedPlaceholder =
+            NSAttributedString(string: "Password", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: Define "Password"

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 10]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        emailSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Email", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: Define string

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 11]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
...
+
+        passwordSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Password", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: Define string

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 12]
```diff
@@ -19,6 +19,14 @@ class RegisterViewController: ViewController {
...
+
+        nameSignUpTextField.attributedPlaceholder =
+            NSAttributedString(string: "Name", attributes: [NSForegroundColorAttributeName: App.Color.placeHolder])
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: Define string

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**